src/did/did.ts

```diff
@@ -16,13 +18,13 @@ export class Did {
    */
   public static validate(did: unknown): void {
     if (typeof did !== 'string') {
-      throw new Error(`DID is not string: ${did}`);
+      throw new DwnError(DwnErrorCode.DidNotValid, `DID is not string: ${did}`);
```

Comment:
Nit: As a coding guideline, we never reuse the same error code. So introduce a new one, perhaps:

```suggestion
      throw new DwnError(DwnErrorCode.DidNotString, `DID is not string: ${did}`);
```